### PR TITLE
Change the way we keep track of user creation time.

### PR DESCRIPTION
**Pull request by royrapoport**
out which users we already know about which we didn't via userhash

We're also moving away from trying to do updates on users we already
know about in the user table.  Instead, we now have a user_created
interface for just the part of this that involves keeping track of the
creation time of a user.

And now that we more reliably have that, we've added csv_reader.py to
upload the Slack's member table, and we're reporting on account creation
date in the user report

### CODE REVIEW

File: user_created.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    def days(self, ts):
+        """
+        return date difference between today and the ts
+        """
+        then = datetime.datetime.fromtimestamp(ts)
+        now = datetime.datetime.now()
+        diff = now - then
+        return diff.days
+
+    def load(self):
```
Comment (jdq22) [inline user_created.py:39]: Will you please write a docblock for this?

Comment (royrapoport) [reply user_created.py:39]: Sorry, for load()?

File: slack_user_report.py [hunk 2]
```diff
@@ -82,7 +95,7 @@ def firstpost(self, uid):
         if not entry:
             return blocks
         m = "By the way, your first-ever message (to the best of our knowledge) was {}, "
-        m += "on {}, {} days ago"
+        m += "on {}, {:,} days ago"
```
Comment (jdq22) [inline slack_user_report.py:98]: 😄

**Review by jdq22 — APPROVED** [review]
I really like the way you implemented the `csv_reader` and `UserCreated`.

I can't think of any, but I'm wondering if there's a better name for `UserCreated`.

Nice work.

File: user_created.py (hunk 1 above)
Comment (jdq22) [reply user_created.py:39]: Yep. Thanks. What you wrote in the docblock makes perfect sense.

Comment (royrapoport) [reply user_created.py:39]: Added a docblock for load()

**Pull request merged**